Dockerfile

```diff
@@ -11,11 +11,26 @@ COPY ca.crt /opt/ca.crt
 ENV CHROME_BIN=chromium-browser NODE_EXTRA_CA_CERTS=/opt/ca.crt
 
 RUN dnf install -y --setopt=tsflags=nodocs \
-    chromium \
-    chromium-headless \
-    findutils \
-    firefox-66.0.2-1.fc30 \
-    ipa-gothic-fonts \
-    libXt \
-    npm \
-    && dnf clean all
+      chromium-73.0.3683.86-2.fc30 \
+      chromium-headless-73.0.3683.86-2.fc30 \ 
+      findutils \
+      firefox-66.0.2-1.fc30 \
+      ipa-gothic-fonts \
+      libXt \
+      npm \
+      wget \
+      unzip\
+      && dnf clean all
+
+# Install webdrivers
+RUN GECKODRIVER_VERSION=`curl https://github.com/mozilla/geckodriver/releases/latest | grep -Po 'v[0-9]+.[0-9]+.[0-9]+'` && \
```

Comment:
Since we're already pinning the versions of chromium, firefox and the chrome driver, it would also make sense to do so here. Let's not try to scrape webpages.

Reply:
I've fixed the geckodriver version to the current one.